packages/office-ui-fabric-react/src/components/Modal/Modal.styles.ts

```diff
@@ -78,7 +78,8 @@ export const getStyles = (props: IModalStyleProps): IModalStyles => {
       classNames.scrollableContent,
       {
         overflowY: 'auto',
-        flexGrow: 1
+        flexGrow: 1,
+        maxHeight: window.innerHeight
```

Comment:
Accessing window here would break server rendered cases where window is not accessible (e.g. SSR tests should have busted here, curious they didn't...)

Plus just READING innerHeight might generate a reflow. See this: https://gist.github.com/paulirish/5d52fb081b3570c81e3a

Can you use `100vh`?  Maybe not because of Safari.

Reply:
@annapablo not sure why SSR tests aren't failing, but maybe you could help take a look.

Reply:
Changed to 100vh with window.innerHeight for iOS. We might be able to just use 100vh, since so far there haven't been any issues with Modal getting clipped on iOS.

Reply:
@dzearing Anna's github userid is @apneer